packages/extension-koni/public/locales/en/translation.json

```diff
@@ -1,5 +1,163 @@
+  "settings.Screen.generalSetting.language.selectLanguage.title": "Language",
+  "settings.Screen.generalSetting.notification.selectPlatform.title": "Notifications",
+  "settings.Screen.generalSetting.Selector.currency.title": "Select a currency",
+  "settings.Screen.generalSetting.Selector.currency.searchPlaceHolder": "Search currency",
+  "settings.security.Screen.changePassword.title": "Change password",
+  "settings.security.Screen.changePassword.bodyTitle": "Change your password",
+  "settings.security.Screen.changePassword.Input.currentPassword.placeHolder": "Current password",
+  "settings.security.Screen.changePassword.FormRules.currentPassword": "Password is required",
+  "settings.security.Screen.changePassword.Input.newPassword.placeHolder": "New password",
+  "settings.security.Screen.changePassword.Input.confirmPassword": "Confirm new password",
+  "settings.security.Screen.changePassword.confirmAgreement": "I understand that SubWallet can’t recover the password.",
+  "settings.security.Screen.changePassword.link.learnMore": "Learn more",
+  "settings.Screen.securitySetting.Selector.autoLock.title": "Auto lock",
+  "settings.Screen.securitySetting.Selector.autoLock.itemLabel": "{{time}} minutes",
+  "settings.Screen.securitySetting.Selector.authPassword.Item.alwaysRequired": "Always required",
+  "settings.Screen.securitySetting.Selector.authPassword.Item.whenNeeded": "When needed",
+  "settings.Screen.manageWebsiteAccess.Filter.substrate": "Substrate dApp",
+  "settings.Screen.manageWebsiteAccess.Filter.ethereum": "Ethereum dApp",
+  "settings.Screen.manageWebsiteAccess.Filter.blocked": "Blocked dApp",
+  "settings.Screen.manageWebsiteAccess.Filter.connected": "Connected dApp",
+  "settings.manageWebsiteAccess.Modal.configuration.Item.forget": "Forget all",
+  "settings.manageWebsiteAccess.Modal.configuration.Item.disconnect": "Disconnect all",
+  "settings.manageWebsiteAccess.Modal.configuration.Item.connect": "Connect all",
+  "settings.manageWebsiteAccess.Modal.accessConfiguration.title": "Access configuration",
+  "settings.Screen.manageNetworks.Filter.evm": "EVM networks",
+  "settings.Screen.manageNetworks.Filter.substrate": "Substrate networks",
+  "settings.Screen.manageNetworks.Filter.custom": "Custom networks",
+  "settings.Screen.manageNetworks.Filter.enabled": "Enabled networks",
+  "settings.Screen.manageNetworks.Filter.disabled": "Disabled networks",
+  "settings.Screen.manageNetworks.importNetworkModal.title": "Import network",
+  "settings.Screen.manageNetworks.importNetworkModal.bodyTitle": "Currently support WSS provider for Substrate networks and HTTP provider for EVM network",
+  "settings.Screen.manageTokens.Filter.enabled": "Enabled tokens",
+  "settings.Screen.manageTokens.Filter.disabled": "Disabled tokens",
+  "settings.Screen.manageTokens.Filter.custom": "Custom tokens",
+  "settings.Screen.manageTokens.importTokens.title": "Import token",
```

Comment:
@dungnguyen-art , change to settings.manageTokens.Screen.importTokens.title, change the related keys too